### ISSUE TITLE
It is not possible to create a translation of an existing post by using the plus sign 

### ISSUE THREAD

**Issue by Chrystll** [polylang/polylang-gutenberg#18]
## Plugins activated
Gutenberg 4.0.0
Polylang Pro 2.4-beta1
Polylang With Gutenberg 0.2.0

## Issue description

It is not possible to create a translation of an existing post by using the plus sign either in the post list or in editing mode.

## Steps to reproduce the issue in editing mode

1. Edit a post in Gutenberg mode editor, here is an English post > then click on the language icon as below to display the Polylang metabox:

![screenshot-trunk-test com-2018 10 22-17-27-39](https://user-images.githubusercontent.com/19824724/47301727-d921c780-d61f-11e8-853d-c5ff1903265d.png)

2. Then just click on the plus sign to create the German translation as here:

![screenshot-trunk-test com-2018 10 22-17-28-38](https://user-images.githubusercontent.com/19824724/47301775-ff476780-d61f-11e8-956f-07ceb4e9cac1.png)

4. After clicking on the plus sign, you obtain a new post in the German language. Then just click on the language icon to display the Polylang icon: in the Polylang metabox, the English post is missing as shown below.

![screenshot-trunk-test com-2018 10 22-17-29-42](https://user-images.githubusercontent.com/19824724/47301906-62d19500-d620-11e8-9205-dd0dd26e02c2.png)

**Comment by Chrystll** [comment]
Currently it is not possible to create a translation of a post. When creating a post translation the url is modified to post-new.php and this breaks the link translation.

## Steps to reproduce the issue:

1. Edit a post in Gutenberg mode editor, here is an English post. Its url is "trunk-test.com/wp-admin/post.php?post=80&action=edit"

2. Then create its German translation by clicking on plus sign. I obtain the new German post with the "trunk-test.com/wp-admin/post-new.php?post_type=post&from_post=80&new_lang=de" url. And the English post is for the moment linked to this new German post as shown here:

![capture du 2018-10-25 14-49-25](https://user-images.githubusercontent.com/19824724/47501593-2db68400-d866-11e8-9c12-c1d9d728752b.png)

3. File a title in this new German post and publish: the url change to "trunk-test.com/wp-admin/post.php?post=82&action=edit" and I loose the translation between the English post and this new German post. See:
![capture du 2018-10-25 14-53-31](https://user-images.githubusercontent.com/19824724/47501604-360ebf00-d866-11e8-872f-1f5404d13f84.png)

**Comment by manooweb** [comment]
Yes because of the autodraft of the block editor which changes the URL with this first saving.
Maybe I have to keep these informations / parameters in the URL in our side for that the metabox works fine in this case.
See also improvment by fetching datas for a new post before displaying the metabox #29

**Comment by manooweb** [comment]
This error is not exactly for the reason I thought above.
It was a regression when I implemented the store and get the title with the context 'edit' parameter is passed to the REST API call and the response was not dispatch in the store
I corrected that and I pushed it but the correction is not complete. In fact, we get only the 'from_post' translated post and not all the translated posts from the original.
I have to work on this.
